.ci/gcb-push-downstream.yml

```diff
@@ -228,7 +224,7 @@ logsBucket: 'gs://cloudbuild-downstream-builder-logs'
 availableSecrets:
   secretManager:
     - versionName: projects/673497134629/secrets/github-classic--repo-workflow/versions/latest
-      env: GITHUB_TOKEN
+      env: GITHUB_TOKEN_CLASSIC
```

Comment:
`vcr-merge` and `magician-check-vcr-cassettes` in this file still reference GITHUB_TOKEN & need to be updated to use GITHUB_TOKEN_CLASSIC instead. The scripts (vcr_merge.sh and check_cassettes respectively) also need to be updated accordingly.